app.js

```diff
@@ -0,0 +1,64 @@
+// jshint esversion:6
+
+const stone = document.getElementById('stone');
+const scissor = document.getElementById('scissor');
+const paper = document.getElementById('paper');
+const result = document.getElementById('result');
+const score = document.querySelector('.score');
+const yourScore = document.querySelector('.your-score');
+const computerScore = document.querySelector('.computer-score');
+const btn = document.querySelectorAll('.btn');
+let yScore = 0;
+let cScore = 0;
+let computerChoice = '';
+
+function computerOption(){
```

Comment:
using arrow functions in es6+ is a good practice.Also, you can replace the callback function with the arrow funtion